src/main/scala/dsls/regex/Program.scala

```diff
@@ -100,12 +105,12 @@ object Program extends App {
   require(number matches "987651234")
 
   /****************************************************************************
-   * TODO: Add the repetition operator for regular expressions
+   * Add the repetition operator for regular expressions
    * 
    * Make it possible to replace the definition of cThree with:
    *    val cThree = 'c'{3}
    ***************************************************************************/
-  val cThree = Concat(Literal('c'), Concat(Literal('c'), Literal('c')))
+  val cThree = 'c'(3)
```

Comment:
I'm not sure it matters, but I think the preferred syntax is `'c'{3}'` (with curly braces).